### ISSUE TITLE
Disabled Publish button is clickable

### ISSUE THREAD

**Issue by mmtr** [WordPress/gutenberg#13014]
**Describe the bug**
As of #11543, `PostPublishButton` doesn't have a `disabled` attribute anymore, but uses `aria-disabled` instead. This is allowing to click on the button, when it shouldn't be possible (for example when the post content is empty).

**To Reproduce**
1. Start creating a new post
2. Note how the "Publish..." button is styled as disabled
3. Click on the "Publish..." button

**Expected behavior**
The Pre-publish checks sidebar shouldn't be opened since the Publish button is disabled. Furthermore, the "Publish" button displayed there is also disabled, but we can still click on it a save the post (note how the date change from "Immediately" to the current date).

**Screenshots**
![image](https://user-images.githubusercontent.com/1233880/50226504-9ff99f80-03a3-11e9-9b3a-8bd01d3b613b.png)

**Additional context**
Issue found on https://github.com/Automattic/wp-calypso/pull/29594 (props to @Copons).

**Comment by swissspidy** [comment]
Duplicate of #11809. PR: #11760.
